buildpipeline/Core-Setup-CrossBuild.json

```diff
@@ -0,0 +1,488 @@
+        "CopyRoot": "$(PB_DockerCopyDest)/corefx",
+        "Contents": "**/*.log",
+        "ArtifactName": "BuildLogs",
+        "ArtifactType": "Container",
+        "TargetPath": "\\\\my\\share\\$(Build.DefinitionName)\\$(Build.BuildNumber)"
+      }
+    },
+    {
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": true,
+      "displayName": "Run docker version",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "docker",
+        "arguments": "version",
+        "workingFolder": "",
+        "failOnStandardError": "false"
+      }
+    }
+  ],
+  "options": [
+    {
+      "enabled": false,
+      "definition": {
+        "id": "7c555368-ca64-4199-add6-9ebaf0b0137d"
+      },
+      "inputs": {
+        "multipliers": "[]",
+        "parallel": "false",
+        "continueOnError": "true",
+        "additionalFields": "{}"
+      }
+    },
+    {
+      "enabled": false,
+      "definition": {
+        "id": "a9db38f9-9fdc-478c-b0f9-464221e58316"
+      },
+      "inputs": {
+        "workItemType": "234347",
+        "assignToRequestor": "true",
+        "additionalFields": "{}"
+      }
+    },
+    {
+      "enabled": false,
+      "definition": {
+        "id": "57578776-4c22-4526-aeb0-86b6da17ee9c"
+      },
+      "inputs": {
+        "additionalFields": "{}"
+      }
+    }
+  ],
+  "variables": {
+    "system.debug": {
+      "value": "false",
+      "allowOverride": true
+    },
+    "PB_ConfigurationGroup": {
+      "value": "Release"
+    },
+    "PB_RepoUrl": {
+      "value": "http://github.com/dotnet/core-setup.git"
+    },
+    "PB_GitDirectory": {
+      "value": "/root/core-setup"
+    },
+    "PB_DockerContainerName": {
+      "value": "$(Build.BuildId)"
+    },
+    "PB_DockerImageName": {
+      "value": "$(PB_DockerRepository):$(PB_DockerTag)"
+    },
+    "PB_DockerRepository": {
+      "value": "chcosta/dotnetcore"
+    },
+    "PB_DockerTag": {
+      "value": "ubuntu1404_cross_prereqs_v1",
+      "allowOverride": true
+    },
+    "PB_CloudDropAccountName": {
+      "value": "dotnetbuildoutput"
+    },
+    "CloudDropAccessToken": {
+      "value": null,
+      "isSecret": true
+    },
+    "OfficialBuildId": {
+      "value": "$(Build.BuildNumber)"
+    },
+    "PB_Label": {
+      "value": "$(Build.BuildNumber)"
+    },
+    "GitHubBranch": {
+      "value": "sni_plus_latestbuildtools"
+    },
+    "PB_Architecture": {
+      "value": "arm"
+    },
+    "SourceVersion": {
+      "value": "HEAD",
+      "allowOverride": true
+    },
+    "PB_DockerCopyDest": {
+      "value": "$(Build.BinariesDirectory)/docker_repo"
+    },
+    "portableLinux": {
+      "value": ""
+    },
+    "CrossToolsetVersion": {
+      "value": "",
+      "allowOverride": true
+    },
+    "SkipUnmount": {
+      "value": "--SkipUnmount"
```

Comment:
@gkhanna79 Just be curious.
Is `--SkipUnmount` (camel case) preferred over `--skipunmount` ?
Because I thought https://github.com/dotnet/core-setup/blob/master/cross/build-rootfs.sh#L28 is just fail-safe code, I've already changed help message of built-rootfs.sh in corefx. If `--SkipUnmount`, i.e. Camel case, is preferred, I will keep  `--SkipUnmount` in help meesage of build-rootfs.sh.